debug:
  publish_registration_clouds: false
  publish_registration_clouds_alt: false
